### PR TITLE
Oscap CLI hammer plugin tests stubs

### PR DESCRIPTION
**Pull request by san7ket**
Added stubs for Oscap-cli hammer plugin
Tier1, Tier2, Tier3 stubs added.
Added the new test_oscap cli test module to rst file , to generated documentation

### CODE REVIEW

**Comment by rochacbruno** [comment]
As you have some empty `@id:` tags you should run locally `make uuid-fix` to get all those filled with new ids.

If you do not want to run this manually every time you can run once `make install-commit-hook` so a new github commit hook will be created and every time you commit those id issues will be fixed and also it will check pep8 issues.

File: tests/foreman/cli/test_oscap.py [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+from robottelo.test import CLITestCase
+
+
+class OpenScapTestCase(CLITestCase):
+    """Tests related to the oscap cli hammer plugin"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_default_scap_content_list_admin(self):
```
Comment (sghai) [inline tests/foreman/cli/test_oscap.py]: `test_positive_list_default_content_with_admin`

File: tests/foreman/cli/test_oscap.py [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+        1. Login to shell from admin account.
+        2. Execute the scap-content command with list as sub-command.
+
+        @Assert: The scap-content are listed.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_default_scap_content_list_viewer_role(self):
```
Comment (sghai) [inline tests/foreman/cli/test_oscap.py]: `test_negative_list_default_content_with_viewer_role`

File: tests/foreman/cli/test_oscap.py [hunk 3]
```diff
@@ -0,0 +1,710 @@
...
+        2. Execute the "scap-content" command with info as sub-command.
+        3. Pass valid "ID" of scap-content as argument.
+
+        @Assert: The info of the scap-content is listed.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_scap_content_info_viewer_role(self):
```
Comment (sghai) [inline tests/foreman/cli/test_oscap.py]: test_negative_view_content_info_with_viewer_role

File: tests/foreman/cli/test_oscap.py [hunk 4]
```diff
@@ -0,0 +1,710 @@
...
+        2. Execute "scap-content" command with "create" as sub-command.
+        3. Pass valid parameters and invalid title.
+
+        @Assert: The scap-content is not created.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_create_scap_content_with_valid_originalfile_name(self):
```
Comment (sghai) [inline tests/foreman/cli/test_oscap.py]: I didn't get `originalfile` name ? Can we give any name ? or is it something specific. ? Lets talk on this once.

Comment (sghai) [inline tests/foreman/cli/test_oscap.py]: test_positive_create_scap_content_with_valid_name ??

File: tests/foreman/cli/test_oscap.py [hunk 5]
```diff
@@ -0,0 +1,710 @@
...
+        2. Execute "policy" command with "info" as sub-command.
+        3. Pass name as the parameter.
+
+        @Assert: The information is displayed.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_update_scap_policy_add_hostgroup(self):
```
Comment (sghai) [inline tests/foreman/cli/test_oscap.py]: test_positive_update_scap_policy_with_hostgroup ?

**Comment by sghai** [comment]
ACK pending few minor comments on test names. 

@kbidarkar : Hey, Since you know this feature very well so could you please review this PR and provide your inputs. Thanks

**Comment by sghai** [comment]
@san7ket : All stubs should have tag `@caseautomation:  notautomated`.

**Comment by kbidarkar** [comment]
ACK, Tests look good and feel we have good coverage for CLI part to start with these.

@san7ket :+1: 

Feel, we should move arf_report tests to t4, as they are quite time consuming and also similar UI tests currently reside there.

**Comment by san7ket** [comment]
@kbidarkar , moved arf_reports related tests to tier4

**Pull request merged**